### PR TITLE
Fix ELRS systems spamming CRSF mode/rate messages

### PR DESCRIPTION
**Pull request by Hwurzburg**
since ELRS does not use the same mode/rate as CRSF, but RC packets are the same, ELRS runs fine on ArduPilot, but users get flooded with this message which is annoying in OSD and GCS....this just stops this one message with an RC_OPTIONS bit (which may have future use for any other issues with ELRS vs CRSF)

flight tested

### CODE REVIEW

File: libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp [hunk 1]
```diff
@@ -165,7 +165,9 @@ void AP_CRSF_Telem::process_rf_mode_changes()
     // report a change in RF mode or a chnage of more than 10Hz if we haven't done so in the last 5s
     if ((now - _telem_last_report_ms > 5000) &&
         (_telem_rf_mode != current_rf_mode || abs(int16_t(_telem_last_avg_rate) - int16_t(_scheduler.avg_packet_rate)) > 25)) {
-        gcs().send_text(MAV_SEVERITY_INFO, "CRSF: RF mode %d, rate is %dHz", (uint8_t)current_rf_mode, _scheduler.avg_packet_rate);
+        if (!rc().surpress_crsf_message()) {
```
Comment (andyp1per) [inline libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp:168]: suppress

File: libraries/RC_Channel/RC_Channel.h [hunk 2]
```diff
@@ -473,6 +473,11 @@ class RC_Channels {
         return _options & uint32_t(Option::ARMING_SKIP_CHECK_RPY);
     }
 
+    bool surpress_crsf_message(void) const {
```
Comment (andyp1per) [inline libraries/RC_Channel/RC_Channel.h:476]: suppress

File: libraries/RC_Channel/RC_Channel.h [hunk 3]
```diff
@@ -531,6 +536,7 @@ class RC_Channels {
         ARMING_SKIP_CHECK_RPY   = (1U << 6), // skip the an arming checks for the roll/pitch/yaw channels
         ALLOW_SWITCH_REV        = (1U << 7), // honor the reversed flag on switches
         CRSF_CUSTOM_TELEMETRY   = (1U << 8), // use passthrough data for crsf telemetry
+        SURPRESS_CRSF_MESSAGE   = (1U << 9), // surpress CRSF mode/rate message for ELRS systems
```
Comment (andyp1per) [inline libraries/RC_Channel/RC_Channel.h:539]: suppress

File: libraries/RC_Channel/RC_Channels_VarInfo.h [hunk 4]
```diff
@@ -89,7 +89,7 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @DisplayName: RC options
     // @Description: RC input options
     // @User: Advanced
-    // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe bit but allow other RC failsafes if setup, 3:FPort Pad, 4:Log RC input bytes, 5:Arming check throttle for 0 input, 6:Skip the arming check for neutral Roll/Pitch/Yay sticks, 7:Allow Switch reverse, 8:Use passthrough for CRSF telemetry
+    // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe bit but allow other RC failsafes if setup, 3:FPort Pad, 4:Log RC input bytes, 5:Arming check throttle for 0 input, 6:Skip the arming check for neutral Roll/Pitch/Yay sticks, 7:Allow Switch reverse, 8:Use passthrough for CRSF telemetry, 9:Surpress CRSF mode/rate message for ELRS systems
```
Comment (andyp1per) [inline libraries/RC_Channel/RC_Channels_VarInfo.h:92]: suppress

**Review by andyp1per — CHANGES_REQUESTED** [review]
Just an important typo to fix

File: libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp]: done

File: libraries/RC_Channel/RC_Channel.h (hunk 2 above)
Comment (Hwurzburg) [reply libraries/RC_Channel/RC_Channel.h]: done

File: libraries/RC_Channel/RC_Channel.h (hunk 3 above)
Comment (Hwurzburg) [reply libraries/RC_Channel/RC_Channel.h]: done

File: libraries/RC_Channel/RC_Channels_VarInfo.h (hunk 4 above)
Comment (Hwurzburg) [reply libraries/RC_Channel/RC_Channels_VarInfo.h]: done

**Review by tridge — APPROVED** [review]

**Pull request merged**